include/spirv-tools/libspirv.h

```diff
@@ -375,6 +379,21 @@ spv_context spvContextCreate(spv_target_env env);
 // Destroys the given context object.
 void spvContextDestroy(spv_context context);
 
+// Creates a Validator options object with default options.
+spv_validator_options spvValidatorOptionsCreate();
+
+// Destroys the given Validator options object.
+void spvValidatorOptionsDestroy(spv_validator_options options);
+
+// Records the maximum number of structure members that is considered valid in
+// the given Validator options object. <options> argument must be non-null.
+void spvValidatorOptionsSetMaxStructMembers(spv_validator_options options,
+                                            const char* limit);
+
+// Returns the maximum number of structure members that is considered valid in
```

Comment:
This does not need to be in the public API.
In general getters on options are not required, and arguably only pollute the interface.